wrappers/python/README.md

```diff
@@ -47,6 +47,16 @@ For development mode installation use
     pip install -e .
 ```
 
+For installing also the requirements needed to run the examples
```

Comment:
Much better. I would drop the also, I don't think it is needed.

Reply:
Done